template.tpl

```diff
@@ -102,6 +102,72 @@ ___TEMPLATE_PARAMETERS___
     "checkboxText": "Enable First Party Cookies",
     "simpleValueType": true,
     "defaultValue": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "advancedMatching",
+    "checkboxText": "Enable Advanced Matching",
+    "simpleValueType": true
+  },
+  {
+    "type": "GROUP",
+    "name": "advancedMatchingGroup",
+    "displayName": "Advanced Matching Parameters",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "advancedMatchingParams",
+        "displayName": "",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Parameter Name",
+            "name": "name",
+            "type": "SELECT",
+            "selectItems": [
+              {
+                "value": "optOut",
+                "displayValue": "Opt Out"
+              },
+              {
+                "value": "email",
+                "displayValue": "Email"
+              },
+              {
+                "value": "aaid",
+                "displayValue": "AAID"
+              },
+              {
+                "value": "idfa",
+                "displayValue": "IDFA"
+              }
+            ],
+            "isUnique": true,
+            "macrosInSelect": false
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Parameter Value",
+            "name": "value",
+            "type": "TEXT",
+            "valueValidators": [
+              {
+                "type": "NON_EMPTY"
+              }
+            ]
+          }
+        ]
+      }
+    ],
+    "enablingConditions": [
+      {
+        "paramName": "advancedMatching",
+        "paramValue": true,
+        "type": "EQUALS"
+      }
+    ],
+    "help": "Advanced Matching Parameters must be pushed to the Data Layer."
```

Comment:
Did marketing approve this help text? I think we may want to make a follow-up ticket to also update https://ads.reddit.com/account/11qnzrqv/conversions#gtm with some additional instructions.